buildkite.yaml

```diff
@@ -5,4 +5,4 @@ steps:
       - "bundle exec rake"
     plugins:
       - docker#v3.7.0:
-          image: ruby:latest
+          image: 445615400570.dkr.ecr.us-east-1.amazonaws.com/ecr-public/docker/library/ruby:3.1.4
```

Comment:
Is this change intentional and what's the motivation behind this it?

Reply:
One of the earlier commits on this branch failed to build due to us hitting [docker cache limits](https://buildkite.com/buildkite/rspec-buildkite-analytics/builds/647#01879745-72d5-4553-9a08-a0b4c608f699). I asked in the [serious eng campfire](https://3.basecamp.com/3453178/buckets/1933834/chats/295546306@6062773808) and James advised changing the image to use one through the pull through cache. 

I can pull this out into a separate PR if you'd prefer?